tests/test_pruner.cpp

```diff
@@ -229,11 +231,13 @@ template <class FT> int test_unpruned()
   pru.enumeration_radius = .85;
   double cost            = pru.single_enum_cost(pr);
   cerr << "Cost per enum " << cost << endl;
-
-  status       = (abs(1 - cost / 3.20e+10) > .02);
+  status |= (abs(1 - cost / 3.20e+10) > .02);
+  status |= std::isnan(cost);
+  cerr << std::isnan(cost) << endl;
```

Comment:
Sounds good to me. It's ultimately your decision in either case, I'm just providing (hopefully useful) feedback.